sdks/python/apache_beam/runners/common.py

```diff
@@ -225,7 +225,7 @@ def process(context):
         else:
           self.dofn_process = lambda context: fn.process(context, *args)
 
-        class CurriedFn(core.DoFn):
+        class CurriedFn(core.OldDoFn):
```

Comment:
How many `OldDoFn` s are left after this change? And why can't we upgrade this one?